inspirehep/modules/workflows/tasks/arxiv.py

```diff
@@ -122,7 +122,7 @@ def arxiv_plot_extract(obj, eng):
             return
 
         for idx, plot in enumerate(plots):
-            obj.files[plot.get('name')] = BytesIO(open(plot.get('url')))
```

Comment:
True! Remember to wrap it around `with open('bla') as bla_fd:` or similar to avoid it :)